### PR TITLE
talk - fix eternal relay-ip

### PR DESCRIPTION
**Pull request by szaimen**
Follow-up to https://github.com/nextcloud/all-in-one/pull/2855

cc @sando38

### CODE REVIEW

File: Containers/talk/start.sh [hunk 1]
```diff
@@ -21,9 +21,6 @@ fi
 set -x
 IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk IN A +short | grep '^[0-9.]\+$' | sort | head -n1)"
 IPv6_ADDRESS_TALK="$(dig nextcloud-aio-talk AAAA +short | grep '^[0-9a-f:]\+$' | sort | head -n1)"
```
Comment (sando38) [inline Containers/talk/start.sh:23]: Are these two always the container IP addresses?

Comment (szaimen) [reply Containers/talk/start.sh:23]: yes

Comment (sando38) [reply Containers/talk/start.sh:23]: Gotcha. In that case, I propose additional changes in the upcoming comment.

**Comment by sando38** [comment]
Follow up (https://github.com/nextcloud/all-in-one/pull/3217#discussion_r1300408723)

I now understand your setup better, it wasn't totally clear for me. As you relay internally between eturnal and your talk component, you do not need a publicly facing IP address as a relay address. I run a similar setup in a kubernetes setup ;)

Therefore, I suggest to eliminate the STUN lookup as well from your Dockerfile, see 2nd option here https://github.com/nextcloud/all-in-one/pull/2855#discussion_r1285715509

**Comment by szaimen** [comment]
Thank you @sando38! :)
Please have a look at https://github.com/nextcloud/all-in-one/pull/3217/commits/2868d45882e9fdf2055a93c65d22019ce8125d8d

**Review by Zoey2936 — APPROVED** [review]
From my side it looks good, but let's wait what @sando38 says

**Comment by sando38** [comment]
> Thank you @sando38! :) Please have a look at [2868d45](https://github.com/nextcloud/all-in-one/commit/2868d45882e9fdf2055a93c65d22019ce8125d8d)

Looks fine from my side.

**Comment by szaimen** [comment]
Thank you both! :)

**Pull request merged**